src/frontend/components/Snackbar/Snackbar.vue

```diff
@@ -0,0 +1,29 @@
+<!-- SnackbarComponent.vue -->
+<template>
+  <v-snackbar v-model="snackbarStore.show" :color="snackbarStore.color">
+    {{ snackbarStore.message }}
+    <template v-slot:actions>
+      <v-btn
+        variant="text"
+        @click="snackbarStore.closeSnackbar"
+      >
+        <v-icon>mdi-close</v-icon>
+      </v-btn>
+    </template>
+  </v-snackbar>
+</template>
+
+<script>
+
+import { useSnackbarStore } from "../../store/snackbar"
```

Comment:
can we use `@store`?

Reply:
Yeah sorry auto-imports don't always do that for me which is dumb. I gotta figure that out. Will push